systemvm/noVNC/include/black.css

```diff
@@ -0,0 +1,71 @@
+/*
+ * noVNC black CSS
+ * Copyright (C) 2012 Joel Martin
+ * Copyright (C) 2013 Samuel Mannehed for Cendio AB
+ * noVNC is licensed under the MPL 2.0 (see LICENSE.txt)
+ * This file is licensed under the 2-Clause BSD license (see LICENSE.txt).
```

Comment:
@syed we might need to discuss with community the license implications of novnc usage within cloudstack codebase/release